org.eclipse.transformer/pom.xml

```diff
@@ -41,6 +41,10 @@
 			<groupId>biz.aQute.bnd</groupId>
 			<artifactId>biz.aQute.bnd.transform</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
bndlib is huge! We do not want to add this dependency. We only want to depend upon `biz.aQute.bnd.transform` which was created to be smaller for transformer use.

Reply:
Removed this dependency